modules/releasing/pages/adjusting-timeouts-resources.adoc

```diff
@@ -62,7 +62,7 @@ spec:
           value: main
         - name: pathInRepo
           value: "<path-to-your-pipeline>"
-    serviceAccountName: appstudio-pipeline
+    serviceAccountName: build-pipeline-$COMPONENT_NAME
```

Comment:
This doesn't seem right. This is for the release pipeline that runs in the releng workspace. `release-service-account` would probably be a good string to use here. See also https://github.com/konflux-ci/release-service-catalog/pull/960

Reply:
In the documentation here, we might not be able to be _this_ prescriptive about the service account name. This name would depend on the configuration of the release pipelines in the managed workspaces, right?

If we want to replace this, I think we would need to be able to override that for any internal documentation.

Reply:
Yes, you're right that there are several different SAs used for release pipelines. But if we agree that having an sample value is good enough here, I think that `release-service-account` would better fit here.